rfcs/0000-community-coordination-hub.md

```diff
@@ -0,0 +1,152 @@
+---
+feature: community_coordination_hub
+start-date: 2020-07-30
+author: David Arnold <dar@xoe.solutions>
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Adopt goals, spirit, concept and execution of [`kubernetes/community`]
+where it _does_ fit [us](https://nixos.org/).
+
+[`kubernetes/community`]: https://github.com/kubernetes/community
+
+# Motivation
+[motivation]: #motivation
+
+We lack an optimum level of _empowerment_ on distinct vital aspects for
+the best [leverage](nav.al/least) on the available resources:
```

Comment:
Nit:

```suggestion
the best [leverage](https://nav.al/least) on the available resources:
```

All links should have `http(s)://`, otherwise, GitHub treats them as relative paths. (e.g., this leads to https://github.com/blaggacao/rfcs/blob/91fd78bae499fca41235386be01a419cfaa6f016/rfcs/nav.al/least)

Reply:
Oh Thank!